torch/quantization/_quantize_script.py

```diff
@@ -82,13 +92,23 @@ def get_scripted_qconfig_dict(qconfig_dict):
 
 def _prepare_script(model, qconfig_dict, is_dynamic):
     _check_is_script_module(model)
+    if not all(map(lambda x : isinstance(x, str), qconfig_dict.keys())):
```

Comment:
Can we run preprocess_qconfig_dict automatically as part of _prepare_script for dynamic quant?

Reply:
answered the question above. We can change the prepare api to accept module instance as well. But that is not very clean since prepare inputs and outputs are scripted module.